test/build/protobuf.cc

```diff
@@ -33,6 +33,7 @@
 
 #include <google/protobuf/descriptor.h>
 #include <google/protobuf/descriptor.pb.h>
+#include <google/protobuf/stubs/strutil.h>
```

Comment:
Yes; that's a private header. Please don't do that. We need to fix the python plugin instead.